cmd/get/get.go

```diff
@@ -14,11 +14,17 @@ func CreateGetCommand() *cobra.Command {
 	}
 
 	getResourcesFuncs := map[string]cmdcore.CommandEntry{
-		"project":    {CmdFunc: getProjectsFunc, Aliases: []string{"projects"}, ProjectDomainNotRequired: true},
-		"task":       {CmdFunc: getTaskFunc, Aliases: []string{"tasks"}},
-		"workflow":   {CmdFunc: getWorkflowFunc, Aliases: []string{"workflows"}},
-		"launchplan": {CmdFunc: getLaunchPlanFunc, Aliases: []string{"launchplans"}},
-		"execution":  {CmdFunc: getExecutionFunc, Aliases: []string{"executions"}},
+		"project": {CmdFunc: getProjectsFunc, Aliases: []string{"projects"}, ProjectDomainNotRequired: true,
+			Short: "Gets project resources",
+			Long:  "Retrieves all the projects"},
+		"task":       {CmdFunc: getTaskFunc, Aliases: []string{"tasks"},Short: "Gets task resources",
+			Long:  "Retrieves all the tasks"},
+		"workflow":   {CmdFunc: getWorkflowFunc, Aliases: []string{"workflows"}, Short: "Gets workflow resources",
+			Long:  "Retrieves all the workflows"},
```

Comment:
you did not move the Long to a variable in every file that containes for example the `getLaunchPlanFunc` that would set a very nice precedent?

Reply:
Done. Moved these

Reply:
Added a few examples.